src/gui/PasswordGeneratorWidget.cpp

```diff
@@ -92,6 +92,7 @@ void PasswordGeneratorWidget::loadSettings()
     m_ui->checkBoxUpper->setChecked(config()->get("generator/UpperCase", true).toBool());
     m_ui->checkBoxNumbers->setChecked(config()->get("generator/Numbers", true).toBool());
     m_ui->checkBoxSpecialChars->setChecked(config()->get("generator/SpecialChars", false).toBool());
+    m_ui->checkBoxExtASCII->setChecked(config()->get("generator/EASCII", false).toBool());
```

Comment:
@louib isn't `Ext` an abbreviation for `Extended`? I will change this to `checkBoxExtendedASCII` if `Ext` don't fit.